//公会修改宣言 公告 徽章
message DceGuildEdit
{
	optional int32	type	= 1;//1修改宣言2修改公告3修改徽章
	optional string content = 2;//宣言或者公告内容
	optional int32	badge	= 3;//徽章
}